src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -199,6 +199,15 @@ void ManageSurfaceHeatBalance(EnergyPlusData &state)
     if (state.dataHeatBalSurfMgr->ManageSurfaceHeatBalancefirstTime) DisplayString(state, "Initializing Surfaces");
     InitSurfaceHeatBalance(state); // Initialize all heat balance related parameters
 
+    // overwrite surface convective and radiative adjustment ratio
```

Comment:
Move this loop to `InitSurfaceHeatBalance` and follow the ` for (int SurfNum = firstSurfWin; SurfNum <= lastSurfWin; ++SurfNum)` zone-surface loop pattern.

Reply:
Thanks. Moved this into InitSurfaceHeatBalance.